### PR TITLE
 Use tokenfield for w:multiSelect

### PR DESCRIPTION
**Pull request by welzfabian**

### CODE REVIEW

**Comment by welzfabian** [comment]
I added the dependencies to wondergem.js, wondergem.css and w:page, I hope that is alright

File: src/main/resources/default/taglib/w/multiSelect.html.pasta [hunk 1]
```diff
@@ -1,45 +1,204 @@
...
+<select id="@id-suggestions-select" style="display: none" multiple="multiple">
+    <i:render name="body"/>
+</select>
+
+<i:if test="optional">
+    <input name="@name" type="hidden" value=""/>
+</i:if>
+
+<script>
+    var initMultiSelect = function (args) {
```
Comment (welzfabian) [inline src/main/resources/default/taglib/w/multiSelect.html.pasta]: I could move that in a separate JS file, any idea where?

File: src/main/resources/default/taglib/w/multiSelect.html.pasta [hunk 2]
```diff
@@ -1,45 +1,195 @@
...
-<i:arg name="help" type="String" default="@i18n(helpKey)" />
-<i:arg name="optional" type="boolean" default="false" />
-<i:arg name="maxItems" type="int" default="0" />
-<i:arg name="required" type="boolean" default="false" />
-<i:arg name="readonly" type="boolean" default="false" />
-<i:arg name="adminOnly" type="boolean" default="false" />
-<i:arg name="forceFullWidth" type="boolean" default="false" />
-<i:arg name="allowCustomEntries" type="boolean" default="false" />
-
-<i:pragma name="inline" value="true" />
```
Comment (sabieber) [inline src/main/resources/default/taglib/w/multiSelect.html.pasta]: why not inline?

File: src/main/resources/default/taglib/w/multiSelect.html.pasta [hunk 3]
```diff
@@ -1,45 +1,195 @@
...
-<i:pragma name="description" value="Renders a mutliselect dropdown field within a Wondergem template" />
```
Comment (sabieber) [inline src/main/resources/default/taglib/w/multiSelect.html.pasta]: Why no description?

File: src/main/resources/default/taglib/w/multiSelect.html.pasta (hunk 1 above)
Comment (sabieber) [reply src/main/resources/default/taglib/w/multiSelect.html.pasta]: Maybe `page-script.html.pasta`?

Comment (andyHa) [reply src/main/resources/default/taglib/w/multiSelect.html.pasta]: yep

**Pull request merged**